assemblers/org.osbuild.qcow2

```diff
@@ -48,12 +39,15 @@ def loop_device(loop_client, image, size, offset=0):
 def main(tree, output_dir, options, loop_client):
     filename = options["filename"]
     root_fs_uuid = options["root_fs_uuid"]
+    size = options["size"]
+
+    if size % 512 != 0:
```

Comment:
The reason for this I think was just that the later tools work in units of sectors, so the excess space would be ignored. I seem to remember getting some warnings at some point, but I cannot reproduce (the code changed a lot).

Maybe add that as a comment (I should have already of course)?

Reply:
Ah, makes sense. I've added a comment.